src/databases/Point3D/avtPoint3DFileFormat.C

```diff
@@ -336,38 +345,54 @@ avtPoint3DFileFormat::ReadData(void)
         EXCEPTION2(InvalidFilesException, filename, "Not ASCII.");
     while (!ifile().eof())
     {
-        // Allow the user to specify "coordflag" in the file.
+        // Allow user to specify "coordflag" in the file.
         if(line[0] == '#')
         {
             if(strncmp(line+1, "coordflag", 9) == 0)
             {
-                debug4 << mName << "Reading coordflag value from file: ";
-                if(strncmp(line+9+2, "xyzv", 4) == 0)
+                debug1 << mName << "Reading coordflag value from file: ";
+                if (strncmp(line+9+2, "xyzv", 4) == 0)
                 {
                     coordFlag = COORDINATE_ORDER_XYZV;
-                    debug4 << "xyzv";
+                    spatialDim = 3;
+                    debug1 << "xyzv";
                 }
-                else if(strncmp(line+9+2, "xyvz", 4) == 0)
+                else if (strncmp(line+9+2, "xyvz", 4) == 0)
                 {
                     coordFlag = COORDINATE_ORDER_XYVZ;
-                    debug4 << "xyvz";
+                    spatialDim = 3;
+                    debug1 << "xyvz";
                 }
-                debug4 << endl;
+                else if (strncmp(line+9+2, "xyv", 3) == 0)
+                {
+                    coordFlag = COORDINATE_ORDER_XYV;
+                    spatialDim = 2;
+                    debug1 << "xyv";
+                }
+                else if (strncmp(line+9+2, "xv", 2) == 0)
+                {
+                    coordFlag = COORDINATE_ORDER_XV;
+                    spatialDim = 1;
+                    debug1 << "xv";
+                }
+                debug1 << endl;
             }
         }
         else
         {
             float a=0, b=0, c=0, d=0;
             int n = sscanf(line, "%f %f %f %f", &a, &b, &c, &d);
-            if (GetStrictMode() && n != 4)
+            if (GetStrictMode() && n != spatialDim+1)
             {
                 EXCEPTION2(InvalidFilesException, filename,
-                           "Bad line in file; less than four values");
+                           "Bad line in file; less than %d values");
```

Comment:
I only see %d in the message, but I don't see where you pass it "spatialDim+1".

Reply:
fixed now